client/src/componenets/Character.js

```diff
@@ -1,19 +1,22 @@
 import React from 'react'
 import Styles from './Character.module.css'
 
-export const Character = (props) => {
-  const data = props.character;
+ const Character = (props) => {
```

Comment:
replace to `const Character = ({ name, id, status, species, origin, image }) => {`